userspace/libsinsp/filterchecks.cpp

```diff
@@ -1873,19 +1873,19 @@ bool sinsp_filter_check_fd::compare(sinsp_evt *evt)
+	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.name", "Name", "The name (excluding the path) of the executable generating the event (task->comm). This field could be populated through a procfs scan (in this case the field is read from /proc/<pid>/status) or through the syscall events stream at runtime (in this case the field is read from the syscalls args)."},
+	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.pname", "Parent Name", "The proc.name (name excluding the path of the parent) of the process generating the event."},
+	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.aname", "Ancestor Name", "List of the proc.name of one of the process ancestors. e.g. proc.aname[1] returns the parent name, proc.aname[2] returns the grandparent name, and so on. proc.aname[0] is the name of the current process. proc.aname without arguments can be used in filters only and matches any of the process ancestors, e.g. proc.aname=bash."},
+	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.args", "Arguments", "The arguments passed on the command line when starting the process generating the event excluding argv[0]. This field could be populated through a procfs scan (in this case the field is read from /proc/<pid>/cmdline) or through the syscall events stream at runtime (in this case the field is read from the syscalls args)."},
 	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.cmdline", "Command Line", "The full command line (proc.name + proc.args) when starting the process generating the event."},
-	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.pcmdline", "Parent Command Line", "The full command line (proc.name + proc.args) of the parent of the process generating the event."},
-	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.acmdline", "Ancestor Command Line", "The full command line (proc.name + proc.args) of one of the process ancestors. e.g. proc.acmdline[1] returns the parent full command line, proc.acmdline[2] returns the grandparent full command line and so on. proc.acmdline[0] is the full command line of the current process. proc.acmdline without arguments can be used in filters only and matches any of the process ancestors, e.g. proc.acmdline contains base64."},
+	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.pcmdline", "Parent Command Line", "The proc.cmdline (full command line (proc.name + proc.args)) of the parent of the process generating the event."},
+	{PT_CHARBUF, EPF_NONE, PF_NA, "proc.acmdline", "Ancestor Command Line", "The list of proc.cmdline (full command line (proc.name + proc.args)) of one of the process ancestors. e.g. proc.acmdline[1] returns the parent full command line, proc.acmdline[2] returns the grandparent full command line and so on. proc.acmdline[0] is the full command line of the current process. proc.acmdline without arguments can be used in filters only and matches any of the process ancestors, e.g. proc.acmdline contains base64."},
```

Comment:
i would remove the `list of` also there

Reply:
Here (and for the others) I think that is better to make clear that is a list... maybe something like this could be a good idea:
```
The full command line (proc.name + proc.args) for a specific process ancestor. You can access different levels of ancestors by using indices. For example, proc.acmdline[1] retrieves the full command line of the parent process, proc.acmdline[2] retrieves the grandparent process, and so on. The current process's full command line can be obtained using proc.acmdline[0].  When used without any arguments, proc.acmdline is applicable only in filters and matches any of the process ancestors. For instance, you can use "proc.acmdline contains base64" to match any process ancestor whose command line contains the term "base64".
```
It's too verbose? WDYT?